app/packages/core/src/components/Actions/ActionsRow.tsx

```diff
@@ -234,7 +234,7 @@ const Selected = ({
       setLoading(e.detail)
     );
 
-  if (samples.size < 1 && !modal) {
+  if (!samples.size || (samples.size < 1 && !modal)) {
```

Comment:
This check unfortunately hides the component in the modal when there are no selected samples. But the button should always be visible in the modal
```suggestion
  if (samples.size < 1 && !modal) {
```